packages/client/src/containers/LandingPage/LandingPage.Container.js

```diff
@@ -19,6 +20,7 @@ export const LandingPage = () => {
     <div className="landing-page-container">
       <ProductList />
       <span>Landing Page</span>
+      <Button label="string" size="small" />
```

Comment:
What is this sneaky button doing here? 🤓

Reply:
I am actually experiment everything with storybook and forgot to delete it 😅 thanks for notice, I will delete that 🙂